src/Chapter1.hs

```diff
@@ -620,8 +625,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = d1 + d2
+  where
+    d1 = mod n 10
+    d2 = div (mod n 100 - d1) 10
```

Comment:
Great!
For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂

Reply:
I just realized that.